Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -7,19 +7,15 @@ import UIKit
 final class PledgePaymentMethodsViewController: UIViewController {
   // MARK: - Properties
 
-  private lazy var applePayButton: PKPaymentButton = { PKPaymentButton() }()
-
-  private lazy var collectionView: UICollectionView = {
-    UICollectionView(frame: .zero, collectionViewLayout: self.layout)
-      |> \.dataSource .~ self
-  }()
-
-  private let layout: UICollectionViewLayout = {
-    UICollectionViewFlowLayout()
-      |> \.scrollDirection .~ .horizontal
-  }()
+  private let viewModel = PledgePaymentMethodsViewModel()
```

Comment:
Can we alphabetize the view model property?

Reply:
For some reason we always add the viewModel property before the other properties. Here I'm just keeping consistency with the rest of the codebase.